rules/app.bzl

```diff
@@ -14,6 +14,7 @@ _IOS_APPLICATION_KWARGS = [
     "provisioning_profile",
     "resources",
     "app_icons",
+    "resources",
```

Comment:
It looks like this is already present on line 15 (which was added somewhat recently, in #120).